src/Sql/dbo/Tables/ApiKey.sql

```diff
@@ -0,0 +1,27 @@
+﻿CREATE TABLE [dbo].[ApiKey] (
+    [Id]               UNIQUEIDENTIFIER,
+    [UserId]           UNIQUEIDENTIFIER NULL,
+    [OrganizationId]   UNIQUEIDENTIFIER NULL,
+    [ServiceAccountId] UNIQUEIDENTIFIER NULL,
+    [ClientSecret]     VARCHAR(30) NOT NULL,
+    [Scope]            NVARCHAR (MAX) NOT NULL,
+    [EncryptedPayload] NVARCHAR (MAX) NOT NULL,
```

Comment:
These seem excessive. It feels possible we'll put expose one of these to userinput someday and we'll want a limit on it. maybe 1Mb to make sure it's never a problem for legit concerns?

Reply:
I changed to 4000, which should be enough for our use cases. There's not really any way to decide on a higher value than 4000 without setting it to max.

Reply:
oh yeah, I forgot about this weird max discrepancy.